shared/src/business/entities/OutboxItem.ts

```diff
@@ -22,7 +22,8 @@ export class OutboxItemClass {
   public leadDocketNumber: string;
   public section: string;
   public assigneeId: string;
-  public trialDate: string;
+  public trialDate?: string;
```

Comment:
trialDate is optional on OutboxItem now?

Reply:
This was more to bring it in line with this field being optional on a WorkItem based on the assumption that an outbox item is simply a completed WorkItem. If that is incorrect I am happy to change it back!

Reply:
Also on further thought, I believe it is optional and allow(null) in the Joi Validation Rules if that makes a difference.

Reply:
Makes sense! Thanks for clarifying that.